packages/lib-user/dev/components/App/App.js

```diff
@@ -98,27 +101,48 @@ function App ({
     }
   }
 
+  const mergedThemes = Object.assign({}, baseTheme, zooTheme)
```

Comment:
A couple notes here:
1. `Object.assign` is outdated syntax and
2. I don't think manual merging is necessary for the Grommet provider anymore. In the [Grommet theming wiki](https://github.com/grommet/grommet/wiki/Grommet-v2-theming-documentation#basics), the example shows a custom theme object passed to the Grommet provider and notes that customization will just merge with the base theme. In app-root's RootLayout component I simply passed the `zooTheme`, so I think that's a strategy to try here too!

https://github.com/zooniverse/front-end-monorepo/blob/04fe35fcfba11d410d8c5cbec893621bfd9f9485/packages/app-root/src/components/RootLayout.js#L29

Reply:
Nice, thanks for noting!